{
    "Analysis_python_files.py": {
        "stats": {
            "total_parameters": 2,
            "parameters_with_annotations": 0
        },
        "error_count": 0,
        "isCompiled": true,
        "errors": []
    },
    "bm_regex_effbot_a245e0361625bbc6129c2b1ec112e9db.py": {
        "stats": {
            "total_parameters": 7,
            "parameters_with_annotations": 0
        },
        "error_count": 1,
        "isCompiled": false,
        "errors": [
            "bm_regex_effbot_a245e0361625bbc6129c2b1ec112e9db.py:82: error: \"Callable[[Any], Any]\" has no attribute \"data\"  [attr-defined]",
            "Found 1 error in 1 file (checked 1 source file)"
        ]
    },
    "bm_regex_v8_a19cb9eda8ce2526c19c5eb844d76145.py": {
        "stats": {
            "total_parameters": 1,
            "parameters_with_annotations": 0
        },
        "error_count": 0,
        "isCompiled": true,
        "errors": []
    },
    "btm_matcher_4f0ccfe480ecd407cea1b164e491bdab.py": {
        "stats": {
            "total_parameters": 12,
            "parameters_with_annotations": 0
        },
        "error_count": 1,
        "isCompiled": false,
        "errors": [
            "btm_matcher_4f0ccfe480ecd407cea1b164e491bdab.py:7: error: No parent module -- cannot perform relative import",
            "Found 1 error in 1 file (errors prevented further checking)"
        ]
    },
    "btm_utils_89de4e6b444fd08def9b8ce6daebb6ce.py": {
        "stats": {
            "total_parameters": 12,
            "parameters_with_annotations": 0
        },
        "error_count": 1,
        "isCompiled": false,
        "errors": [
            "btm_utils_89de4e6b444fd08def9b8ce6daebb6ce.py:3: error: No parent module -- cannot perform relative import",
            "Found 1 error in 1 file (errors prevented further checking)"
        ]
    },
    "conf_da9965b984331586a98ccf31b5479f12.py": {
        "stats": {
            "total_parameters": 0,
            "parameters_with_annotations": 0
        },
        "error_count": 0,
        "isCompiled": true,
        "errors": []
    },
    "conv_1bc4e8a8f8f49e5d7327949ef65b3e34.py": {
        "stats": {
            "total_parameters": 8,
            "parameters_with_annotations": 0
        },
        "error_count": 0,
        "isCompiled": true,
        "errors": []
    },
    "driver_3aabcaf38ec34819d53840a6d8011e52.py": {
        "stats": {
            "total_parameters": 30,
            "parameters_with_annotations": 0
        },
        "error_count": 1,
        "isCompiled": false,
        "errors": [
            "driver_3aabcaf38ec34819d53840a6d8011e52.py:10: error: No parent module -- cannot perform relative import",
            "Found 1 error in 1 file (errors prevented further checking)"
        ]
    },
    "fixer_base_85dc36bb56186b3c5d4a5ff7bd806974.py": {
        "stats": {
            "total_parameters": 30,
            "parameters_with_annotations": 0
        },
        "error_count": 1,
        "isCompiled": false,
        "errors": [
            "fixer_base_85dc36bb56186b3c5d4a5ff7bd806974.py:5: error: No parent module -- cannot perform relative import",
            "Found 1 error in 1 file (errors prevented further checking)"
        ]
    },
    "fixer_util_3abd089b71f391b9fbcc1279a5103b06.py": {
        "stats": {
            "total_parameters": 54,
            "parameters_with_annotations": 0
        },
        "error_count": 1,
        "isCompiled": false,
        "errors": [
            "fixer_util_3abd089b71f391b9fbcc1279a5103b06.py:6: error: No parent module -- cannot perform relative import",
            "Found 1 error in 1 file (errors prevented further checking)"
        ]
    },
    "fix_apply_b16e3ffb9d7076892b1bf5bf775adbe3.py": {
        "stats": {
            "total_parameters": 3,
            "parameters_with_annotations": 0
        },
        "error_count": 2,
        "isCompiled": false,
        "errors": [
            "fix_apply_b16e3ffb9d7076892b1bf5bf775adbe3.py:3: error: No parent module -- cannot perform relative import",
            "fix_apply_b16e3ffb9d7076892b1bf5bf775adbe3.py:5: error: No parent module -- cannot perform relative import",
            "Found 2 errors in 1 file (errors prevented further checking)"
        ]
    },
    "fix_asserts_4f68a58c5bfe9899342f37ae6be10efb.py": {
        "stats": {
            "total_parameters": 3,
            "parameters_with_annotations": 0
        },
        "error_count": 2,
        "isCompiled": false,
        "errors": [
            "fix_asserts_4f68a58c5bfe9899342f37ae6be10efb.py:3: error: Relative import climbs too many namespaces  [misc]",
            "fix_asserts_4f68a58c5bfe9899342f37ae6be10efb.py:4: error: Relative import climbs too many namespaces  [misc]",
            "Found 2 errors in 1 file (checked 1 source file)"
        ]
    },
    "fix_basestring_84e7b7d66f65f34af7dad68be8649969.py": {
        "stats": {
            "total_parameters": 3,
            "parameters_with_annotations": 0
        },
        "error_count": 1,
        "isCompiled": false,
        "errors": [
            "fix_basestring_84e7b7d66f65f34af7dad68be8649969.py:3: error: No parent module -- cannot perform relative import",
            "Found 1 error in 1 file (errors prevented further checking)"
        ]
    },
    "fix_buffer_0297747e2fb863eb527f0296b4e04c1d.py": {
        "stats": {
            "total_parameters": 3,
            "parameters_with_annotations": 0
        },
        "error_count": 1,
        "isCompiled": false,
        "errors": [
            "fix_buffer_0297747e2fb863eb527f0296b4e04c1d.py:3: error: No parent module -- cannot perform relative import",
            "Found 1 error in 1 file (errors prevented further checking)"
        ]
    },
    "fix_dict_26321b67f43cc9e91f518d3409058c17.py": {
        "stats": {
            "total_parameters": 6,
            "parameters_with_annotations": 0
        },
        "error_count": 4,
        "isCompiled": false,
        "errors": [
            "fix_dict_26321b67f43cc9e91f518d3409058c17.py:3: error: No parent module -- cannot perform relative import",
            "fix_dict_26321b67f43cc9e91f518d3409058c17.py:4: error: No parent module -- cannot perform relative import",
            "fix_dict_26321b67f43cc9e91f518d3409058c17.py:5: error: No parent module -- cannot perform relative import",
            "fix_dict_26321b67f43cc9e91f518d3409058c17.py:7: error: No parent module -- cannot perform relative import",
            "Found 4 errors in 1 file (errors prevented further checking)"
        ]
    },
    "fix_except_dc2a3518b6572d9df36c7c6a9e7d185c.py": {
        "stats": {
            "total_parameters": 4,
            "parameters_with_annotations": 0
        },
        "error_count": 2,
        "isCompiled": false,
        "errors": [
            "fix_except_dc2a3518b6572d9df36c7c6a9e7d185c.py:3: error: No parent module -- cannot perform relative import",
            "fix_except_dc2a3518b6572d9df36c7c6a9e7d185c.py:5: error: No parent module -- cannot perform relative import",
            "Found 2 errors in 1 file (errors prevented further checking)"
        ]
    },
    "fix_execfile_23c0aea8521d5905ef574849a1199c42.py": {
        "stats": {
            "total_parameters": 3,
            "parameters_with_annotations": 0
        },
        "error_count": 1,
        "isCompiled": false,
        "errors": [
            "fix_execfile_23c0aea8521d5905ef574849a1199c42.py:3: error: No parent module -- cannot perform relative import",
            "Found 1 error in 1 file (errors prevented further checking)"
        ]
    },
    "fix_exec_c2495e4b98c855ef8e4bb6b499835916.py": {
        "stats": {
            "total_parameters": 3,
            "parameters_with_annotations": 0
        },
        "error_count": 1,
        "isCompiled": false,
        "errors": [
            "fix_exec_c2495e4b98c855ef8e4bb6b499835916.py:3: error: No parent module -- cannot perform relative import",
            "Found 1 error in 1 file (errors prevented further checking)"
        ]
    },
    "fix_exitfunc_4c0e023d6233f367f6bbb22f72abff19.py": {
        "stats": {
            "total_parameters": 7,
            "parameters_with_annotations": 0
        },
        "error_count": 0,
        "isCompiled": true,
        "errors": []
    },
    "fix_filter_0d6c2a2f1d6dc89e448a17d7d1859c3a.py": {
        "stats": {
            "total_parameters": 3,
            "parameters_with_annotations": 0
        },
        "error_count": 1,
        "isCompiled": false,
        "errors": [
            "fix_filter_0d6c2a2f1d6dc89e448a17d7d1859c3a.py:3: error: No parent module -- cannot perform relative import",
            "Found 1 error in 1 file (errors prevented further checking)"
        ]
    },
    "fix_funcattrs_b0872da875aad669f1282398989d9c89.py": {
        "stats": {
            "total_parameters": 3,
            "parameters_with_annotations": 0
        },
        "error_count": 1,
        "isCompiled": false,
        "errors": [
            "fix_funcattrs_b0872da875aad669f1282398989d9c89.py:3: error: No parent module -- cannot perform relative import",
            "Found 1 error in 1 file (errors prevented further checking)"
        ]
    },
    "fix_future_23a687d0e9b1a05680b4c5ab0decab24.py": {
        "stats": {
            "total_parameters": 3,
            "parameters_with_annotations": 0
        },
        "error_count": 1,
        "isCompiled": false,
        "errors": [
            "fix_future_23a687d0e9b1a05680b4c5ab0decab24.py:3: error: No parent module -- cannot perform relative import",
            "Found 1 error in 1 file (errors prevented further checking)"
        ]
    },
    "fix_getcwdu_4c7a9a4fe905d6e1ec1a1f745fc07e92.py": {
        "stats": {
            "total_parameters": 3,
            "parameters_with_annotations": 0
        },
        "error_count": 1,
        "isCompiled": false,
        "errors": [
            "fix_getcwdu_4c7a9a4fe905d6e1ec1a1f745fc07e92.py:3: error: No parent module -- cannot perform relative import",
            "Found 1 error in 1 file (errors prevented further checking)"
        ]
    },
    "fix_has_key_3d19bb5809047e970a7a7694f342c22f.py": {
        "stats": {
            "total_parameters": 3,
            "parameters_with_annotations": 0
        },
        "error_count": 2,
        "isCompiled": false,
        "errors": [
            "fix_has_key_3d19bb5809047e970a7a7694f342c22f.py:3: error: No parent module -- cannot perform relative import",
            "fix_has_key_3d19bb5809047e970a7a7694f342c22f.py:4: error: No parent module -- cannot perform relative import",
            "Found 2 errors in 1 file (errors prevented further checking)"
        ]
    },
    "fix_idioms_f2fc0fac4f3e5f15360819a753c8ce8d.py": {
        "stats": {
            "total_parameters": 14,
            "parameters_with_annotations": 0
        },
        "error_count": 1,
        "isCompiled": false,
        "errors": [
            "fix_idioms_f2fc0fac4f3e5f15360819a753c8ce8d.py:3: error: No parent module -- cannot perform relative import",
            "Found 1 error in 1 file (errors prevented further checking)"
        ]
    },
    "fix_imports2_990f0acae87e29ae9492418a41754ee9.py": {
        "stats": {
            "total_parameters": 0,
            "parameters_with_annotations": 0
        },
        "error_count": 1,
        "isCompiled": false,
        "errors": [
            "fix_imports2_990f0acae87e29ae9492418a41754ee9.py:3: error: No parent module -- cannot perform relative import",
            "Found 1 error in 1 file (errors prevented further checking)"
        ]
    },
    "fix_imports_c928b33522d728da5c7972ba72bd5d6b.py": {
        "stats": {
            "total_parameters": 12,
            "parameters_with_annotations": 0
        },
        "error_count": 1,
        "isCompiled": false,
        "errors": [
            "fix_imports_c928b33522d728da5c7972ba72bd5d6b.py:3: error: No parent module -- cannot perform relative import",
            "Found 1 error in 1 file (errors prevented further checking)"
        ]
    },
    "fix_import_f23058f61e15a73e76de69f135bf5f6f.py": {
        "stats": {
            "total_parameters": 9,
            "parameters_with_annotations": 0
        },
        "error_count": 1,
        "isCompiled": false,
        "errors": [
            "fix_import_f23058f61e15a73e76de69f135bf5f6f.py:3: error: No parent module -- cannot perform relative import",
            "Found 1 error in 1 file (errors prevented further checking)"
        ]
    },
    "fix_input_dfe10ba83564ecc0d92ce3fe3f7469f6.py": {
        "stats": {
            "total_parameters": 3,
            "parameters_with_annotations": 0
        },
        "error_count": 2,
        "isCompiled": false,
        "errors": [
            "fix_input_dfe10ba83564ecc0d92ce3fe3f7469f6.py:3: error: No parent module -- cannot perform relative import",
            "fix_input_dfe10ba83564ecc0d92ce3fe3f7469f6.py:5: error: No parent module -- cannot perform relative import",
            "Found 2 errors in 1 file (errors prevented further checking)"
        ]
    },
    "fix_intern_293f735a6b785b784192f0be1636168a.py": {
        "stats": {
            "total_parameters": 3,
            "parameters_with_annotations": 0
        },
        "error_count": 1,
        "isCompiled": false,
        "errors": [
            "fix_intern_293f735a6b785b784192f0be1636168a.py:3: error: No parent module -- cannot perform relative import",
            "Found 1 error in 1 file (errors prevented further checking)"
        ]
    },
    "fix_isinstance_ddf41b92f0feef3fdda51bb5aa571b20.py": {
        "stats": {
            "total_parameters": 3,
            "parameters_with_annotations": 0
        },
        "error_count": 1,
        "isCompiled": false,
        "errors": [
            "fix_isinstance_ddf41b92f0feef3fdda51bb5aa571b20.py:3: error: No parent module -- cannot perform relative import",
            "Found 1 error in 1 file (errors prevented further checking)"
        ]
    },
    "fix_itertools_4f695aa383ccd1d83a24563d5dfb7466.py": {
        "stats": {
            "total_parameters": 3,
            "parameters_with_annotations": 0
        },
        "error_count": 1,
        "isCompiled": false,
        "errors": [
            "fix_itertools_4f695aa383ccd1d83a24563d5dfb7466.py:3: error: No parent module -- cannot perform relative import",
            "Found 1 error in 1 file (errors prevented further checking)"
        ]
    },
    "fix_itertools_imports_1ff0945ccfa029ee1e633d3b0c38dca8.py": {
        "stats": {
            "total_parameters": 3,
            "parameters_with_annotations": 0
        },
        "error_count": 1,
        "isCompiled": false,
        "errors": [
            "fix_itertools_imports_1ff0945ccfa029ee1e633d3b0c38dca8.py:8: error: Not all arguments converted during string formatting  [str-format]",
            "Found 1 error in 1 file (checked 1 source file)"
        ]
    },
    "fix_long_c9fe2d241de1cc017034e4f7bc2217cb.py": {
        "stats": {
            "total_parameters": 3,
            "parameters_with_annotations": 0
        },
        "error_count": 0,
        "isCompiled": true,
        "errors": []
    },
    "fix_map_923c607ffa53d4554c8b2f301b449ede.py": {
        "stats": {
            "total_parameters": 3,
            "parameters_with_annotations": 0
        },
        "error_count": 1,
        "isCompiled": false,
        "errors": [
            "fix_map_923c607ffa53d4554c8b2f301b449ede.py:4: error: No parent module -- cannot perform relative import",
            "Found 1 error in 1 file (errors prevented further checking)"
        ]
    },
    "fix_metaclass_c0624dc69dbc6726b21bb4af3b427e71.py": {
        "stats": {
            "total_parameters": 11,
            "parameters_with_annotations": 0
        },
        "error_count": 1,
        "isCompiled": false,
        "errors": [
            "fix_metaclass_c0624dc69dbc6726b21bb4af3b427e71.py:3: error: No parent module -- cannot perform relative import",
            "Found 1 error in 1 file (errors prevented further checking)"
        ]
    },
    "fix_methodattrs_6a42825e2099caece0bd5f7c288ff26d.py": {
        "stats": {
            "total_parameters": 3,
            "parameters_with_annotations": 0
        },
        "error_count": 1,
        "isCompiled": false,
        "errors": [
            "fix_methodattrs_6a42825e2099caece0bd5f7c288ff26d.py:3: error: No parent module -- cannot perform relative import",
            "Found 1 error in 1 file (errors prevented further checking)"
        ]
    },
    "fix_next_804cb524bc47688ff19276d35ba9a378.py": {
        "stats": {
            "total_parameters": 10,
            "parameters_with_annotations": 0
        },
        "error_count": 1,
        "isCompiled": false,
        "errors": [
            "fix_next_804cb524bc47688ff19276d35ba9a378.py:5: error: No parent module -- cannot perform relative import",
            "Found 1 error in 1 file (errors prevented further checking)"
        ]
    },
    "fix_ne_99cb992e21ba5c9cdd4ed2465485a813.py": {
        "stats": {
            "total_parameters": 5,
            "parameters_with_annotations": 0
        },
        "error_count": 2,
        "isCompiled": false,
        "errors": [
            "fix_ne_99cb992e21ba5c9cdd4ed2465485a813.py:3: error: No parent module -- cannot perform relative import",
            "fix_ne_99cb992e21ba5c9cdd4ed2465485a813.py:5: error: No parent module -- cannot perform relative import",
            "Found 2 errors in 1 file (errors prevented further checking)"
        ]
    },
    "fix_nonzero_1692b34cd0de1184d67c1e62cc8d6ec7.py": {
        "stats": {
            "total_parameters": 3,
            "parameters_with_annotations": 0
        },
        "error_count": 1,
        "isCompiled": false,
        "errors": [
            "fix_nonzero_1692b34cd0de1184d67c1e62cc8d6ec7.py:3: error: No parent module -- cannot perform relative import",
            "Found 1 error in 1 file (errors prevented further checking)"
        ]
    },
    "fix_numliterals_98271bbcfbd011fb007cd3a1ad98359d.py": {
        "stats": {
            "total_parameters": 5,
            "parameters_with_annotations": 0
        },
        "error_count": 1,
        "isCompiled": false,
        "errors": [
            "fix_numliterals_98271bbcfbd011fb007cd3a1ad98359d.py:4: error: No parent module -- cannot perform relative import",
            "Found 1 error in 1 file (errors prevented further checking)"
        ]
    },
    "fix_operator_41eec7e7f93f16bad1cfd1a8ce899fbd.py": {
        "stats": {
            "total_parameters": 38,
            "parameters_with_annotations": 0
        },
        "error_count": 0,
        "isCompiled": true,
        "errors": []
    },
    "fix_paren_353b6cf540d829d4cb86fcdf6b1c3685.py": {
        "stats": {
            "total_parameters": 3,
            "parameters_with_annotations": 0
        },
        "error_count": 1,
        "isCompiled": false,
        "errors": [
            "fix_paren_353b6cf540d829d4cb86fcdf6b1c3685.py:3: error: No parent module -- cannot perform relative import",
            "Found 1 error in 1 file (errors prevented further checking)"
        ]
    },
    "fix_print_aca65419859f1ac0135d2ad32457f8e8.py": {
        "stats": {
            "total_parameters": 7,
            "parameters_with_annotations": 0
        },
        "error_count": 3,
        "isCompiled": false,
        "errors": [
            "fix_print_aca65419859f1ac0135d2ad32457f8e8.py:3: error: No parent module -- cannot perform relative import",
            "fix_print_aca65419859f1ac0135d2ad32457f8e8.py:4: error: No parent module -- cannot perform relative import",
            "fix_print_aca65419859f1ac0135d2ad32457f8e8.py:6: error: No parent module -- cannot perform relative import",
            "Found 3 errors in 1 file (errors prevented further checking)"
        ]
    },
    "fix_raise_495345fa6dfbfe2a576ace6d7c035c51.py": {
        "stats": {
            "total_parameters": 3,
            "parameters_with_annotations": 0
        },
        "error_count": 2,
        "isCompiled": false,
        "errors": [
            "fix_raise_495345fa6dfbfe2a576ace6d7c035c51.py:3: error: No parent module -- cannot perform relative import",
            "fix_raise_495345fa6dfbfe2a576ace6d7c035c51.py:5: error: No parent module -- cannot perform relative import",
            "Found 2 errors in 1 file (errors prevented further checking)"
        ]
    },
    "fix_raw_input_b30be6e6f6b371c5d0cc4238916aaac7.py": {
        "stats": {
            "total_parameters": 3,
            "parameters_with_annotations": 0
        },
        "error_count": 1,
        "isCompiled": false,
        "errors": [
            "fix_raw_input_b30be6e6f6b371c5d0cc4238916aaac7.py:3: error: No parent module -- cannot perform relative import",
            "Found 1 error in 1 file (errors prevented further checking)"
        ]
    },
    "fix_reduce_4024a50c8ce9def06d8aabec27d605d4.py": {
        "stats": {
            "total_parameters": 3,
            "parameters_with_annotations": 0
        },
        "error_count": 0,
        "isCompiled": true,
        "errors": []
    },
    "fix_reload_c7113cf30125ff3fd4e963b1761da463.py": {
        "stats": {
            "total_parameters": 3,
            "parameters_with_annotations": 0
        },
        "error_count": 1,
        "isCompiled": false,
        "errors": [
            "fix_reload_c7113cf30125ff3fd4e963b1761da463.py:3: error: No parent module -- cannot perform relative import",
            "Found 1 error in 1 file (errors prevented further checking)"
        ]
    },
    "fix_renames_1eb568db217ff15c2b4004ccd75dd169.py": {
        "stats": {
            "total_parameters": 6,
            "parameters_with_annotations": 0
        },
        "error_count": 1,
        "isCompiled": false,
        "errors": [
            "fix_renames_1eb568db217ff15c2b4004ccd75dd169.py:3: error: No parent module -- cannot perform relative import",
            "Found 1 error in 1 file (errors prevented further checking)"
        ]
    },
    "fix_repr_4c23954f65d7efb15afeef1f4931d3d3.py": {
        "stats": {
            "total_parameters": 3,
            "parameters_with_annotations": 0
        },
        "error_count": 1,
        "isCompiled": false,
        "errors": [
            "fix_repr_4c23954f65d7efb15afeef1f4931d3d3.py:3: error: No parent module -- cannot perform relative import",
            "Found 1 error in 1 file (errors prevented further checking)"
        ]
    },
    "fix_set_literal_4c0da782a04c8b621da519425739911b.py": {
        "stats": {
            "total_parameters": 3,
            "parameters_with_annotations": 0
        },
        "error_count": 0,
        "isCompiled": true,
        "errors": []
    },
    "fix_standarderror_6354be6de21e5d0ae8243cc615985cae.py": {
        "stats": {
            "total_parameters": 3,
            "parameters_with_annotations": 0
        },
        "error_count": 1,
        "isCompiled": false,
        "errors": [
            "fix_standarderror_6354be6de21e5d0ae8243cc615985cae.py:3: error: No parent module -- cannot perform relative import",
            "Found 1 error in 1 file (errors prevented further checking)"
        ]
    },
    "fix_sys_exc_38efdf62222966e271d0328ae81e4b2f.py": {
        "stats": {
            "total_parameters": 3,
            "parameters_with_annotations": 0
        },
        "error_count": 1,
        "isCompiled": false,
        "errors": [
            "fix_sys_exc_38efdf62222966e271d0328ae81e4b2f.py:3: error: No parent module -- cannot perform relative import",
            "Found 1 error in 1 file (errors prevented further checking)"
        ]
    },
    "fix_throw_9223bab1596f455827d0ca24d2cb4b8a.py": {
        "stats": {
            "total_parameters": 3,
            "parameters_with_annotations": 0
        },
        "error_count": 2,
        "isCompiled": false,
        "errors": [
            "fix_throw_9223bab1596f455827d0ca24d2cb4b8a.py:3: error: No parent module -- cannot perform relative import",
            "fix_throw_9223bab1596f455827d0ca24d2cb4b8a.py:5: error: No parent module -- cannot perform relative import",
            "Found 2 errors in 1 file (errors prevented further checking)"
        ]
    },
    "fix_tuple_params_08235f2c848bdeb79dabf448ca7b343e.py": {
        "stats": {
            "total_parameters": 15,
            "parameters_with_annotations": 0
        },
        "error_count": 2,
        "isCompiled": false,
        "errors": [
            "fix_tuple_params_08235f2c848bdeb79dabf448ca7b343e.py:3: error: No parent module -- cannot perform relative import",
            "fix_tuple_params_08235f2c848bdeb79dabf448ca7b343e.py:5: error: No parent module -- cannot perform relative import",
            "Found 2 errors in 1 file (errors prevented further checking)"
        ]
    },
    "fix_types_6de0a2197b78900ec88ac8b7b3545921.py": {
        "stats": {
            "total_parameters": 3,
            "parameters_with_annotations": 0
        },
        "error_count": 1,
        "isCompiled": false,
        "errors": [
            "fix_types_6de0a2197b78900ec88ac8b7b3545921.py:3: error: No parent module -- cannot perform relative import",
            "Found 1 error in 1 file (errors prevented further checking)"
        ]
    },
    "fix_unicode_ea7cfd666cad0427f50e6bb2fe1a479d.py": {
        "stats": {
            "total_parameters": 6,
            "parameters_with_annotations": 0
        },
        "error_count": 1,
        "isCompiled": false,
        "errors": [
            "fix_unicode_ea7cfd666cad0427f50e6bb2fe1a479d.py:4: error: No parent module -- cannot perform relative import",
            "Found 1 error in 1 file (errors prevented further checking)"
        ]
    },
    "fix_urllib_e4836c50388897297cf89c61693daed3.py": {
        "stats": {
            "total_parameters": 15,
            "parameters_with_annotations": 0
        },
        "error_count": 0,
        "isCompiled": true,
        "errors": []
    },
    "fix_ws_comma_3c3b2f49eeb26b8d75079583accc8d3c.py": {
        "stats": {
            "total_parameters": 3,
            "parameters_with_annotations": 0
        },
        "error_count": 2,
        "isCompiled": false,
        "errors": [
            "fix_ws_comma_3c3b2f49eeb26b8d75079583accc8d3c.py:3: error: No parent module -- cannot perform relative import",
            "fix_ws_comma_3c3b2f49eeb26b8d75079583accc8d3c.py:5: error: No parent module -- cannot perform relative import",
            "Found 2 errors in 1 file (errors prevented further checking)"
        ]
    },
    "fix_xrange_b7597d1a2dcf484c9def32299bab0389.py": {
        "stats": {
            "total_parameters": 17,
            "parameters_with_annotations": 0
        },
        "error_count": 2,
        "isCompiled": false,
        "errors": [
            "fix_xrange_b7597d1a2dcf484c9def32299bab0389.py:3: error: No parent module -- cannot perform relative import",
            "fix_xrange_b7597d1a2dcf484c9def32299bab0389.py:5: error: No parent module -- cannot perform relative import",
            "Found 2 errors in 1 file (errors prevented further checking)"
        ]
    },
    "fix_xreadlines_336d1f82c56a878207062958ce346079.py": {
        "stats": {
            "total_parameters": 3,
            "parameters_with_annotations": 0
        },
        "error_count": 1,
        "isCompiled": false,
        "errors": [
            "fix_xreadlines_336d1f82c56a878207062958ce346079.py:3: error: No parent module -- cannot perform relative import",
            "Found 1 error in 1 file (errors prevented further checking)"
        ]
    },
    "fix_zip_8deafd1b0091d77b3a4b65e272fcdb39.py": {
        "stats": {
            "total_parameters": 3,
            "parameters_with_annotations": 0
        },
        "error_count": 1,
        "isCompiled": false,
        "errors": [
            "fix_zip_8deafd1b0091d77b3a4b65e272fcdb39.py:3: error: No parent module -- cannot perform relative import",
            "Found 1 error in 1 file (errors prevented further checking)"
        ]
    },
    "generate_data_7422c52c22b8c15ab496c22f64433879.py": {
        "stats": {
            "total_parameters": 0,
            "parameters_with_annotations": 0
        },
        "error_count": 4,
        "isCompiled": false,
        "errors": [
            "generate_data_7422c52c22b8c15ab496c22f64433879.py:4: error: Library stubs not installed for \"toml\"  [import]",
            "generate_data_7422c52c22b8c15ab496c22f64433879.py:4: note: Hint: \"python3 -m pip install types-toml\"",
            "generate_data_7422c52c22b8c15ab496c22f64433879.py:4: note: (or run \"mypy --install-types\" to install all missing stub packages)",
            "generate_data_7422c52c22b8c15ab496c22f64433879.py:4: note: See https://mypy.readthedocs.io/en/stable/running_mypy.html#missing-imports",
            "Found 1 error in 1 file (checked 1 source file)"
        ]
    },
    "Generate_no_type_version_147101a8e98a675af12167080b175470.py": {
        "stats": {
            "total_parameters": 7,
            "parameters_with_annotations": 0
        },
        "error_count": 0,
        "isCompiled": true,
        "errors": []
    },
    "grammar_0ee226e126c4ae4d1f4233f1a436b1ad.py": {
        "stats": {
            "total_parameters": 9,
            "parameters_with_annotations": 0
        },
        "error_count": 1,
        "isCompiled": false,
        "errors": [
            "grammar_0ee226e126c4ae4d1f4233f1a436b1ad.py:4: error: No parent module -- cannot perform relative import",
            "Found 1 error in 1 file (errors prevented further checking)"
        ]
    },
    "literals_51e85ba6bc607dc435c2756415e18998.py": {
        "stats": {
            "total_parameters": 2,
            "parameters_with_annotations": 0
        },
        "error_count": 0,
        "isCompiled": true,
        "errors": []
    },
    "main_18e06710863c6ff46857d6e5f5e3193a.py": {
        "stats": {
            "total_parameters": 27,
            "parameters_with_annotations": 0
        },
        "error_count": 1,
        "isCompiled": false,
        "errors": [
            "main_18e06710863c6ff46857d6e5f5e3193a.py:10: error: No parent module -- cannot perform relative import",
            "Found 1 error in 1 file (errors prevented further checking)"
        ]
    },
    "parse_16d2dea206b527e5e6f274f72e54d079.py": {
        "stats": {
            "total_parameters": 30,
            "parameters_with_annotations": 0
        },
        "error_count": 1,
        "isCompiled": false,
        "errors": [
            "parse_16d2dea206b527e5e6f274f72e54d079.py:3: error: No parent module -- cannot perform relative import",
            "Found 1 error in 1 file (errors prevented further checking)"
        ]
    },
    "patcomp_39f9f5e83f63a35080e3487619196bab.py": {
        "stats": {
            "total_parameters": 18,
            "parameters_with_annotations": 0
        },
        "error_count": 2,
        "isCompiled": false,
        "errors": [
            "patcomp_39f9f5e83f63a35080e3487619196bab.py:6: error: No parent module -- cannot perform relative import",
            "patcomp_39f9f5e83f63a35080e3487619196bab.py:7: error: No parent module -- cannot perform relative import",
            "Found 2 errors in 1 file (errors prevented further checking)"
        ]
    },
    "pgen_eba48e8b0455077a205d1dda049755b8.py": {
        "stats": {
            "total_parameters": 55,
            "parameters_with_annotations": 0
        },
        "error_count": 1,
        "isCompiled": false,
        "errors": [
            "pgen_eba48e8b0455077a205d1dda049755b8.py:2: error: No parent module -- cannot perform relative import",
            "Found 1 error in 1 file (errors prevented further checking)"
        ]
    },
    "pygram_a5391fb5dc3ff905dab45ee155e42ff9.py": {
        "stats": {
            "total_parameters": 2,
            "parameters_with_annotations": 0
        },
        "error_count": 1,
        "isCompiled": false,
        "errors": [
            "pygram_a5391fb5dc3ff905dab45ee155e42ff9.py:6: error: No parent module -- cannot perform relative import",
            "Found 1 error in 1 file (errors prevented further checking)"
        ]
    },
    "pyspecific_5869ad860916164ee81485b5281bc3c9.py": {
        "stats": {
            "total_parameters": 37,
            "parameters_with_annotations": 0
        },
        "error_count": 7,
        "isCompiled": false,
        "errors": [
            "pyspecific_5869ad860916164ee81485b5281bc3c9.py:3: error: Library stubs not installed for \"docutils\"  [import]",
            "pyspecific_5869ad860916164ee81485b5281bc3c9.py:4: error: Library stubs not installed for \"docutils.parsers.rst\"  [import]",
            "pyspecific_5869ad860916164ee81485b5281bc3c9.py:5: error: Library stubs not installed for \"docutils.utils\"  [import]",
            "pyspecific_5869ad860916164ee81485b5281bc3c9.py:14: error: Library stubs not installed for \"docutils.parsers.rst.states\"  [import]",
            "pyspecific_5869ad860916164ee81485b5281bc3c9.py:14: note: Hint: \"python3 -m pip install types-docutils\"",
            "pyspecific_5869ad860916164ee81485b5281bc3c9.py:14: note: (or run \"mypy --install-types\" to install all missing stub packages)",
            "pyspecific_5869ad860916164ee81485b5281bc3c9.py:14: note: See https://mypy.readthedocs.io/en/stable/running_mypy.html#missing-imports",
            "Found 4 errors in 1 file (checked 1 source file)"
        ]
    },
    "pytree_b5c9498d48307fc20f4016a12676f355.py": {
        "stats": {
            "total_parameters": 122,
            "parameters_with_annotations": 0
        },
        "error_count": 2,
        "isCompiled": false,
        "errors": [
            "pytree_b5c9498d48307fc20f4016a12676f355.py:7: error: Need type annotation for \"_type_reprs\" (hint: \"_type_reprs: Dict[<type>, <type>] = ...\")  [var-annotated]",
            "pytree_b5c9498d48307fc20f4016a12676f355.py:36: error: Incompatible types in assignment (expression has type \"None\", base class \"object\" defined the type as \"Callable[[object], int]\")  [assignment]",
            "Found 2 errors in 1 file (checked 1 source file)"
        ]
    },
    "refactor_ba19b34abce9dfcf909394be809bc04a.py": {
        "stats": {
            "total_parameters": 87,
            "parameters_with_annotations": 0
        },
        "error_count": 2,
        "isCompiled": false,
        "errors": [
            "refactor_ba19b34abce9dfcf909394be809bc04a.py:14: error: No parent module -- cannot perform relative import",
            "refactor_ba19b34abce9dfcf909394be809bc04a.py:15: error: No parent module -- cannot perform relative import",
            "Found 2 errors in 1 file (errors prevented further checking)"
        ]
    },
    "run_benchmark_05192c0c8d5a8999a8c094b2ecf40473.py": {
        "stats": {
            "total_parameters": 63,
            "parameters_with_annotations": 0
        },
        "error_count": 0,
        "isCompiled": true,
        "errors": []
    },
    "run_benchmark_079afee64b38f3e4ec499d6009906612.py": {
        "stats": {
            "total_parameters": 2,
            "parameters_with_annotations": 0
        },
        "error_count": 0,
        "isCompiled": true,
        "errors": []
    },
    "run_benchmark_083fc82df7d9596c7ee0e92cd5ab86ea.py": {
        "stats": {
            "total_parameters": 1,
            "parameters_with_annotations": 0
        },
        "error_count": 0,
        "isCompiled": true,
        "errors": []
    },
    "run_benchmark_0ae4d57c5b86320985ca055ab916dace.py": {
        "stats": {
            "total_parameters": 5,
            "parameters_with_annotations": 0
        },
        "error_count": 1,
        "isCompiled": false,
        "errors": [
            "run_benchmark_0ae4d57c5b86320985ca055ab916dace.py:30: error: Incompatible types in assignment (expression has type \"List[str]\", variable has type \"Tuple[Any]\")  [assignment]",
            "Found 1 error in 1 file (checked 1 source file)"
        ]
    },
    "run_benchmark_101bb04648591f38cbb42eaf6aba3288.py": {
        "stats": {
            "total_parameters": 83,
            "parameters_with_annotations": 0
        },
        "error_count": 0,
        "isCompiled": true,
        "errors": []
    },
    "run_benchmark_14b1aad8ef40880d34ce019b57e1dd81.py": {
        "stats": {
            "total_parameters": 8,
            "parameters_with_annotations": 0
        },
        "error_count": 0,
        "isCompiled": true,
        "errors": []
    },
    "run_benchmark_17dc7effd52ebd41c89a16fb84ba976e.py": {
        "stats": {
            "total_parameters": 17,
            "parameters_with_annotations": 0
        },
        "error_count": 0,
        "isCompiled": true,
        "errors": []
    },
    "run_benchmark_1be7503fa8723e91833eb80f31228737.py": {
        "stats": {
            "total_parameters": 2,
            "parameters_with_annotations": 0
        },
        "error_count": 0,
        "isCompiled": true,
        "errors": []
    },
    "run_benchmark_1efa63cfa255cc633c18f7dce4818351.py": {
        "stats": {
            "total_parameters": 3,
            "parameters_with_annotations": 0
        },
        "error_count": 0,
        "isCompiled": true,
        "errors": []
    },
    "run_benchmark_21026ff8f81b122e6c9cfbf8c29a27cb.py": {
        "stats": {
            "total_parameters": 5,
            "parameters_with_annotations": 0
        },
        "error_count": 0,
        "isCompiled": true,
        "errors": []
    },
    "run_benchmark_26727fba89826eeafa7ccc4e1e90c284.py": {
        "stats": {
            "total_parameters": 4,
            "parameters_with_annotations": 0
        },
        "error_count": 0,
        "isCompiled": true,
        "errors": []
    },
    "run_benchmark_268e2f631afb81a04deab49d7cf3fb46.py": {
        "stats": {
            "total_parameters": 4,
            "parameters_with_annotations": 0
        },
        "error_count": 1,
        "isCompiled": false,
        "errors": [
            "run_benchmark_268e2f631afb81a04deab49d7cf3fb46.py:47: error: Incompatible types in assignment (expression has type \"Tuple[str, str, str, str]\", variable has type \"Tuple[Any]\")  [assignment]",
            "Found 1 error in 1 file (checked 1 source file)"
        ]
    },
    "run_benchmark_293989a6d5e7f93e90fe279a3c2f7db9.py": {
        "stats": {
            "total_parameters": 5,
            "parameters_with_annotations": 0
        },
        "error_count": 0,
        "isCompiled": true,
        "errors": []
    },
    "run_benchmark_2d2b8b8374ccfc0f944485cd5da03ebc.py": {
        "stats": {
            "total_parameters": 1,
            "parameters_with_annotations": 0
        },
        "error_count": 0,
        "isCompiled": true,
        "errors": []
    },
    "run_benchmark_34494d3bb60355e7d1a04faddcfffc1d.py": {
        "stats": {
            "total_parameters": 8,
            "parameters_with_annotations": 0
        },
        "error_count": 0,
        "isCompiled": true,
        "errors": []
    },
    "run_benchmark_35477730ea622662cb6c95ff69a68e99.py": {
        "stats": {
            "total_parameters": 25,
            "parameters_with_annotations": 0
        },
        "error_count": 7,
        "isCompiled": false,
        "errors": [
            "run_benchmark_35477730ea622662cb6c95ff69a68e99.py:165: error: Incompatible types in assignment (expression has type \"None\", target has type Module)  [assignment]",
            "run_benchmark_35477730ea622662cb6c95ff69a68e99.py:173: error: All conditional function variants must have identical signatures  [misc]",
            "run_benchmark_35477730ea622662cb6c95ff69a68e99.py:173: note: Original:",
            "run_benchmark_35477730ea622662cb6c95ff69a68e99.py:173: note:     def import_module(name: str, package: Optional[str] = ...) -> Module",
            "run_benchmark_35477730ea622662cb6c95ff69a68e99.py:173: note: Redefinition:",
            "run_benchmark_35477730ea622662cb6c95ff69a68e99.py:173: note:     def import_module(module_name: Any) -> Any",
            "run_benchmark_35477730ea622662cb6c95ff69a68e99.py:197: error: Incompatible types in assignment (expression has type \"List[str]\", variable has type \"Tuple[Any]\")  [assignment]",
            "Found 3 errors in 1 file (checked 1 source file)"
        ]
    },
    "run_benchmark_41759fdf0401f89c962cc7b114aa3e42.py": {
        "stats": {
            "total_parameters": 1,
            "parameters_with_annotations": 0
        },
        "error_count": 0,
        "isCompiled": true,
        "errors": []
    },
    "run_benchmark_432ac9707ee16dffc2f2fe3f1c6bf76c.py": {
        "stats": {
            "total_parameters": 20,
            "parameters_with_annotations": 0
        },
        "error_count": 1,
        "isCompiled": false,
        "errors": [
            "run_benchmark_432ac9707ee16dffc2f2fe3f1c6bf76c.py:163: error: Incompatible types in assignment (expression has type \"None\", target has type Module)  [assignment]",
            "Found 1 error in 1 file (checked 1 source file)"
        ]
    },
    "run_benchmark_4dc0acdd1240a804f34da8d966a065e1.py": {
        "stats": {
            "total_parameters": 7,
            "parameters_with_annotations": 0
        },
        "error_count": 0,
        "isCompiled": true,
        "errors": []
    },
    "run_benchmark_54d6e19c1743b019edc613e74d8321ba.py": {
        "stats": {
            "total_parameters": 115,
            "parameters_with_annotations": 0
        },
        "error_count": 7,
        "isCompiled": false,
        "errors": [
            "run_benchmark_54d6e19c1743b019edc613e74d8321ba.py:45: error: Incompatible types in assignment (expression has type \"Strength\", variable has type \"None\")  [assignment]",
            "run_benchmark_54d6e19c1743b019edc613e74d8321ba.py:46: error: Incompatible types in assignment (expression has type \"Strength\", variable has type \"None\")  [assignment]",
            "run_benchmark_54d6e19c1743b019edc613e74d8321ba.py:47: error: Incompatible types in assignment (expression has type \"Strength\", variable has type \"None\")  [assignment]",
            "run_benchmark_54d6e19c1743b019edc613e74d8321ba.py:48: error: Incompatible types in assignment (expression has type \"Strength\", variable has type \"None\")  [assignment]",
            "run_benchmark_54d6e19c1743b019edc613e74d8321ba.py:49: error: Incompatible types in assignment (expression has type \"Strength\", variable has type \"None\")  [assignment]",
            "run_benchmark_54d6e19c1743b019edc613e74d8321ba.py:50: error: Incompatible types in assignment (expression has type \"Strength\", variable has type \"None\")  [assignment]",
            "run_benchmark_54d6e19c1743b019edc613e74d8321ba.py:51: error: Incompatible types in assignment (expression has type \"Strength\", variable has type \"None\")  [assignment]",
            "Found 7 errors in 1 file (checked 1 source file)"
        ]
    },
    "run_benchmark_5f738045810f54167c085e1c7e0b35b8.py": {
        "stats": {
            "total_parameters": 3,
            "parameters_with_annotations": 0
        },
        "error_count": 0,
        "isCompiled": true,
        "errors": []
    },
    "run_benchmark_6021b120af897a98b208540880d31fa1.py": {
        "stats": {
            "total_parameters": 7,
            "parameters_with_annotations": 0
        },
        "error_count": 0,
        "isCompiled": true,
        "errors": []
    },
    "run_benchmark_64e361fd1586e053709bd8ec787fefaf.py": {
        "stats": {
            "total_parameters": 4,
            "parameters_with_annotations": 0
        },
        "error_count": 0,
        "isCompiled": true,
        "errors": []
    },
    "run_benchmark_64f067051f53db8fe8e69329b3e087ed.py": {
        "stats": {
            "total_parameters": 1,
            "parameters_with_annotations": 0
        },
        "error_count": 0,
        "isCompiled": true,
        "errors": []
    },
    "run_benchmark_6c16d199d98991cfd04b86f116962777.py": {
        "stats": {
            "total_parameters": 1,
            "parameters_with_annotations": 0
        },
        "error_count": 0,
        "isCompiled": true,
        "errors": []
    },
    "run_benchmark_735d8e18ff99fd95e3e85b3a045a96ff.py": {
        "stats": {
            "total_parameters": 73,
            "parameters_with_annotations": 0
        },
        "error_count": 0,
        "isCompiled": true,
        "errors": []
    },
    "run_benchmark_7465a3358d6be0a308ff0dba1c019896.py": {
        "stats": {
            "total_parameters": 2,
            "parameters_with_annotations": 0
        },
        "error_count": 0,
        "isCompiled": true,
        "errors": []
    },
    "run_benchmark_74e46dc7e24ad277a3ffd520c33499c2.py": {
        "stats": {
            "total_parameters": 4,
            "parameters_with_annotations": 0
        },
        "error_count": 6,
        "isCompiled": false,
        "errors": [
            "run_benchmark_74e46dc7e24ad277a3ffd520c33499c2.py:9: error: Variable \"run_benchmark_74e46dc7e24ad277a3ffd520c33499c2.Base\" is not valid as a type  [valid-type]",
            "run_benchmark_74e46dc7e24ad277a3ffd520c33499c2.py:9: note: See https://mypy.readthedocs.io/en/stable/common_issues.html#variables-vs-type-aliases",
            "run_benchmark_74e46dc7e24ad277a3ffd520c33499c2.py:9: error: Invalid base class \"Base\"  [misc]",
            "run_benchmark_74e46dc7e24ad277a3ffd520c33499c2.py:14: error: Variable \"run_benchmark_74e46dc7e24ad277a3ffd520c33499c2.Base\" is not valid as a type  [valid-type]",
            "run_benchmark_74e46dc7e24ad277a3ffd520c33499c2.py:14: note: See https://mypy.readthedocs.io/en/stable/common_issues.html#variables-vs-type-aliases",
            "run_benchmark_74e46dc7e24ad277a3ffd520c33499c2.py:14: error: Invalid base class \"Base\"  [misc]",
            "Found 4 errors in 1 file (checked 1 source file)"
        ]
    },
    "run_benchmark_79b0563155ae7c19376dd17a256bd4c9.py": {
        "stats": {
            "total_parameters": 4,
            "parameters_with_annotations": 0
        },
        "error_count": 0,
        "isCompiled": true,
        "errors": []
    },
    "run_benchmark_7e52e6cf6b6ac0c8c20602f1713d3f9e.py": {
        "stats": {
            "total_parameters": 0,
            "parameters_with_annotations": 0
        },
        "error_count": 0,
        "isCompiled": true,
        "errors": []
    },
    "run_benchmark_9196e0e1a8569bc4d538f9e91f44380b.py": {
        "stats": {
            "total_parameters": 7,
            "parameters_with_annotations": 0
        },
        "error_count": 0,
        "isCompiled": true,
        "errors": []
    },
    "run_benchmark_9473d82c1ad213cf726be7d21540b9f3.py": {
        "stats": {
            "total_parameters": 6,
            "parameters_with_annotations": 0
        },
        "error_count": 0,
        "isCompiled": true,
        "errors": []
    },
    "run_benchmark_958ac12d9a928de9aad6c1b814c51ef7.py": {
        "stats": {
            "total_parameters": 51,
            "parameters_with_annotations": 0
        },
        "error_count": 0,
        "isCompiled": true,
        "errors": []
    },
    "run_benchmark_95edf135ddae7bd47dbd5e98374098f5.py": {
        "stats": {
            "total_parameters": 1,
            "parameters_with_annotations": 0
        },
        "error_count": 0,
        "isCompiled": true,
        "errors": []
    },
    "run_benchmark_97b7db16323aeacce0598574c19d7dc4.py": {
        "stats": {
            "total_parameters": 4,
            "parameters_with_annotations": 0
        },
        "error_count": 0,
        "isCompiled": true,
        "errors": []
    },
    "run_benchmark_a245e0361625bbc6129c2b1ec112e9db.py": {
        "stats": {
            "total_parameters": 7,
            "parameters_with_annotations": 0
        },
        "error_count": 1,
        "isCompiled": false,
        "errors": [
            "run_benchmark_a245e0361625bbc6129c2b1ec112e9db.py:82: error: \"Callable[[Any], Any]\" has no attribute \"data\"  [attr-defined]",
            "Found 1 error in 1 file (checked 1 source file)"
        ]
    },
    "run_benchmark_a28c6e02f54a08e0b801c4930e06d67b.py": {
        "stats": {
            "total_parameters": 0,
            "parameters_with_annotations": 0
        },
        "error_count": 1,
        "isCompiled": false,
        "errors": [
            "run_benchmark_a28c6e02f54a08e0b801c4930e06d67b.py:15: error: \"None\" has no attribute \"set\"  [attr-defined]",
            "Found 1 error in 1 file (checked 1 source file)"
        ]
    },
    "run_benchmark_a33097bd5241761d2d3d046b36d9afd4.py": {
        "stats": {
            "total_parameters": 10,
            "parameters_with_annotations": 0
        },
        "error_count": 0,
        "isCompiled": true,
        "errors": []
    },
    "run_benchmark_a55d3dd13b512cfe214145c5e92afdc6.py": {
        "stats": {
            "total_parameters": 3,
            "parameters_with_annotations": 0
        },
        "error_count": 1,
        "isCompiled": false,
        "errors": [
            "run_benchmark_a55d3dd13b512cfe214145c5e92afdc6.py:5: error: Need type annotation for \"EMPTY\"  [var-annotated]",
            "Found 1 error in 1 file (checked 1 source file)"
        ]
    },
    "run_benchmark_a98f596fc7b09c623910e5db7e8f8ffc.py": {
        "stats": {
            "total_parameters": 0,
            "parameters_with_annotations": 0
        },
        "error_count": 0,
        "isCompiled": true,
        "errors": []
    },
    "run_benchmark_ad565b18eba62edc0cc5a53f755fbc29.py": {
        "stats": {
            "total_parameters": 7,
            "parameters_with_annotations": 0
        },
        "error_count": 0,
        "isCompiled": true,
        "errors": []
    },
    "run_benchmark_b26b9b306d7c5987b898c17b5ef45275.py": {
        "stats": {
            "total_parameters": 5,
            "parameters_with_annotations": 0
        },
        "error_count": 0,
        "isCompiled": true,
        "errors": []
    },
    "run_benchmark_b89518a404d01c8afa20b38d197e3f9a.py": {
        "stats": {
            "total_parameters": 114,
            "parameters_with_annotations": 0
        },
        "error_count": 8,
        "isCompiled": false,
        "errors": [
            "run_benchmark_b89518a404d01c8afa20b38d197e3f9a.py:71: error: \"Type[Vector]\" has no attribute \"ZERO\"  [attr-defined]",
            "run_benchmark_b89518a404d01c8afa20b38d197e3f9a.py:72: error: \"Type[Vector]\" has no attribute \"RIGHT\"  [attr-defined]",
            "run_benchmark_b89518a404d01c8afa20b38d197e3f9a.py:73: error: \"Type[Vector]\" has no attribute \"UP\"  [attr-defined]",
            "run_benchmark_b89518a404d01c8afa20b38d197e3f9a.py:74: error: \"Type[Vector]\" has no attribute \"OUT\"  [attr-defined]",
            "run_benchmark_b89518a404d01c8afa20b38d197e3f9a.py:75: error: \"Type[Vector]\" has no attribute \"RIGHT\"  [attr-defined]",
            "run_benchmark_b89518a404d01c8afa20b38d197e3f9a.py:75: error: \"Type[Vector]\" has no attribute \"UP\"  [attr-defined]",
            "run_benchmark_b89518a404d01c8afa20b38d197e3f9a.py:76: error: \"Type[Vector]\" has no attribute \"UP\"  [attr-defined]",
            "run_benchmark_b89518a404d01c8afa20b38d197e3f9a.py:165: error: \"Type[Point]\" has no attribute \"ZERO\"  [attr-defined]",
            "Found 8 errors in 1 file (checked 1 source file)"
        ]
    },
    "run_benchmark_ba302be588b03433d3833330ef334694.py": {
        "stats": {
            "total_parameters": 1,
            "parameters_with_annotations": 0
        },
        "error_count": 0,
        "isCompiled": true,
        "errors": []
    },
    "run_benchmark_bbaface73b5271d284816da26ab45a2e.py": {
        "stats": {
            "total_parameters": 3,
            "parameters_with_annotations": 0
        },
        "error_count": 5,
        "isCompiled": false,
        "errors": [
            "run_benchmark_bbaface73b5271d284816da26ab45a2e.py:5: error: Library stubs not installed for \"docutils\"  [import]",
            "run_benchmark_bbaface73b5271d284816da26ab45a2e.py:9: error: Library stubs not installed for \"docutils.utils.math.math2html\"  [import]",
            "run_benchmark_bbaface73b5271d284816da26ab45a2e.py:9: note: Hint: \"python3 -m pip install types-docutils\"",
            "run_benchmark_bbaface73b5271d284816da26ab45a2e.py:9: note: (or run \"mypy --install-types\" to install all missing stub packages)",
            "run_benchmark_bbaface73b5271d284816da26ab45a2e.py:9: note: See https://mypy.readthedocs.io/en/stable/running_mypy.html#missing-imports",
            "Found 2 errors in 1 file (checked 1 source file)"
        ]
    },
    "run_benchmark_c193048bf6974d42edd1a8e1d38d51fa.py": {
        "stats": {
            "total_parameters": 16,
            "parameters_with_annotations": 0
        },
        "error_count": 2,
        "isCompiled": false,
        "errors": [
            "run_benchmark_c193048bf6974d42edd1a8e1d38d51fa.py:117: error: Incompatible types in assignment (expression has type \"None\", variable has type \"int\")  [assignment]",
            "run_benchmark_c193048bf6974d42edd1a8e1d38d51fa.py:118: error: Incompatible types in assignment (expression has type \"None\", variable has type \"Tuple[List[int], int, int, int]\")  [assignment]",
            "Found 2 errors in 1 file (checked 1 source file)"
        ]
    },
    "run_benchmark_ca073e899d60ce995a3685d97078e2ed.py": {
        "stats": {
            "total_parameters": 2,
            "parameters_with_annotations": 0
        },
        "error_count": 0,
        "isCompiled": true,
        "errors": []
    },
    "run_benchmark_cca3b64b92d026f4feb24fcec90fe159.py": {
        "stats": {
            "total_parameters": 8,
            "parameters_with_annotations": 0
        },
        "error_count": 0,
        "isCompiled": true,
        "errors": []
    },
    "run_benchmark_ce0e39f2be42224684270a194a2fe3b4.py": {
        "stats": {
            "total_parameters": 18,
            "parameters_with_annotations": 0
        },
        "error_count": 0,
        "isCompiled": true,
        "errors": []
    },
    "run_benchmark_d0531dbcb196b002efdeaa7a81b1fd9c.py": {
        "stats": {
            "total_parameters": 0,
            "parameters_with_annotations": 0
        },
        "error_count": 0,
        "isCompiled": true,
        "errors": []
    },
    "run_benchmark_d55e5927075139a1ac0cc6dbe7fee59b.py": {
        "stats": {
            "total_parameters": 3,
            "parameters_with_annotations": 0
        },
        "error_count": 0,
        "isCompiled": true,
        "errors": []
    },
    "run_benchmark_d5780ee905aac0d8545a6b5d3e1c3af6.py": {
        "stats": {
            "total_parameters": 13,
            "parameters_with_annotations": 0
        },
        "error_count": 1,
        "isCompiled": false,
        "errors": [
            "run_benchmark_d5780ee905aac0d8545a6b5d3e1c3af6.py:39: error: Incompatible types in assignment (expression has type \"Callable[[Any, Any], Any]\", variable has type \"Callable[[Union[str, bytes, PathLike[str], PathLike[bytes]], Union[str, bytes, PathLike[str], PathLike[bytes]], DefaultNamedArg(Optional[int], 'src_dir_fd'), DefaultNamedArg(Optional[int], 'dst_dir_fd')], None]\")  [assignment]",
            "Found 1 error in 1 file (checked 1 source file)"
        ]
    },
    "run_benchmark_d7485f1540815ff9846e47c93bb7b997.py": {
        "stats": {
            "total_parameters": 85,
            "parameters_with_annotations": 0
        },
        "error_count": 0,
        "isCompiled": true,
        "errors": []
    },
    "run_benchmark_db768e7124854f442d014d84437038ed.py": {
        "stats": {
            "total_parameters": 85,
            "parameters_with_annotations": 0
        },
        "error_count": 0,
        "isCompiled": true,
        "errors": []
    },
    "run_benchmark_dc3dd40536a1de9023f8266cf6fdbfe8.py": {
        "stats": {
            "total_parameters": 49,
            "parameters_with_annotations": 0
        },
        "error_count": 0,
        "isCompiled": true,
        "errors": []
    },
    "run_benchmark_dff26d535b0e89b50097c808066750ec.py": {
        "stats": {
            "total_parameters": 31,
            "parameters_with_annotations": 0
        },
        "error_count": 0,
        "isCompiled": true,
        "errors": []
    },
    "run_benchmark_e13886742d29f7347d4ce5fd3df9e7ac.py": {
        "stats": {
            "total_parameters": 1,
            "parameters_with_annotations": 0
        },
        "error_count": 0,
        "isCompiled": true,
        "errors": []
    },
    "run_benchmark_e36e06e08b165fd948b9fec76a0d5acf.py": {
        "stats": {
            "total_parameters": 12,
            "parameters_with_annotations": 0
        },
        "error_count": 1,
        "isCompiled": false,
        "errors": [
            "run_benchmark_e36e06e08b165fd948b9fec76a0d5acf.py:98: error: Incompatible types in assignment (expression has type \"List[str]\", variable has type \"Tuple[Any]\")  [assignment]",
            "Found 1 error in 1 file (checked 1 source file)"
        ]
    },
    "run_benchmark_e40666714496fa08885bd7c6883de76b.py": {
        "stats": {
            "total_parameters": 25,
            "parameters_with_annotations": 0
        },
        "error_count": 7,
        "isCompiled": false,
        "errors": [
            "run_benchmark_e40666714496fa08885bd7c6883de76b.py:10: error: All protocol members must have explicitly declared types  [misc]",
            "run_benchmark_e40666714496fa08885bd7c6883de76b.py:15: error: All protocol members must have explicitly declared types  [misc]",
            "run_benchmark_e40666714496fa08885bd7c6883de76b.py:16: error: All protocol members must have explicitly declared types  [misc]",
            "run_benchmark_e40666714496fa08885bd7c6883de76b.py:17: error: All protocol members must have explicitly declared types  [misc]",
            "run_benchmark_e40666714496fa08885bd7c6883de76b.py:18: error: All protocol members must have explicitly declared types  [misc]",
            "run_benchmark_e40666714496fa08885bd7c6883de76b.py:19: error: All protocol members must have explicitly declared types  [misc]",
            "run_benchmark_e40666714496fa08885bd7c6883de76b.py:50: error: All protocol members must have explicitly declared types  [misc]",
            "Found 7 errors in 1 file (checked 1 source file)"
        ]
    },
    "run_benchmark_e77a04005a6abbca76058ef014ab41a6.py": {
        "stats": {
            "total_parameters": 4,
            "parameters_with_annotations": 0
        },
        "error_count": 1,
        "isCompiled": false,
        "errors": [
            "run_benchmark_e77a04005a6abbca76058ef014ab41a6.py:6: error: Incompatible types in assignment (expression has type \"None\", target has type Module)  [assignment]",
            "Found 1 error in 1 file (checked 1 source file)"
        ]
    },
    "run_benchmark_e7f5416c8fddb15d9ca4fdd9c195a6ee.py": {
        "stats": {
            "total_parameters": 1,
            "parameters_with_annotations": 0
        },
        "error_count": 0,
        "isCompiled": true,
        "errors": []
    },
    "run_benchmark_e93163c78613c4410d2fe21b2a81f7a2.py": {
        "stats": {
            "total_parameters": 10,
            "parameters_with_annotations": 0
        },
        "error_count": 0,
        "isCompiled": true,
        "errors": []
    },
    "run_benchmark_ed36b868873c9ca9cb835088629996b5.py": {
        "stats": {
            "total_parameters": 7,
            "parameters_with_annotations": 0
        },
        "error_count": 0,
        "isCompiled": true,
        "errors": []
    },
    "run_benchmark_ef50c0004e4c63b45751afdedd20f922.py": {
        "stats": {
            "total_parameters": 7,
            "parameters_with_annotations": 0
        },
        "error_count": 0,
        "isCompiled": true,
        "errors": []
    },
    "run_benchmark_f4779276baee1e691111c58f163c03a0.py": {
        "stats": {
            "total_parameters": 4,
            "parameters_with_annotations": 0
        },
        "error_count": 0,
        "isCompiled": true,
        "errors": []
    },
    "run_benchmark_f4909fc93f2b77fbaa7a1a62f396ad97.py": {
        "stats": {
            "total_parameters": 82,
            "parameters_with_annotations": 0
        },
        "error_count": 1,
        "isCompiled": false,
        "errors": [
            "run_benchmark_f4909fc93f2b77fbaa7a1a62f396ad97.py:348: error: Incompatible types in assignment (expression has type \"List[str]\", variable has type \"Tuple[Any]\")  [assignment]",
            "Found 1 error in 1 file (checked 1 source file)"
        ]
    },
    "run_benchmark_fe7e80f88187eee851b45d280f1d907a.py": {
        "stats": {
            "total_parameters": 0,
            "parameters_with_annotations": 0
        },
        "error_count": 0,
        "isCompiled": true,
        "errors": []
    },
    "run_benchmark_ff55a410a7f31c4a3ea3646528a81cb1.py": {
        "stats": {
            "total_parameters": 2,
            "parameters_with_annotations": 0
        },
        "error_count": 0,
        "isCompiled": true,
        "errors": []
    },
    "Run_raw_mypy.py": {
        "stats": {
            "total_parameters": 3,
            "parameters_with_annotations": 0
        },
        "error_count": 0,
        "isCompiled": true,
        "errors": []
    },
    "tokenize_b7662d3f564d80fcd8b53af809f8752a.py": {
        "stats": {
            "total_parameters": 22,
            "parameters_with_annotations": 0
        },
        "error_count": 1,
        "isCompiled": false,
        "errors": [
            "tokenize_b7662d3f564d80fcd8b53af809f8752a.py:8: error: No parent module -- cannot perform relative import",
            "Found 1 error in 1 file (errors prevented further checking)"
        ]
    },
    "token_5adf5a279650e10ecd71aed939a26b65.py": {
        "stats": {
            "total_parameters": 3,
            "parameters_with_annotations": 0
        },
        "error_count": 0,
        "isCompiled": true,
        "errors": []
    },
    "Unix_file_Path.py": {
        "stats": {
            "total_parameters": 1,
            "parameters_with_annotations": 0
        },
        "error_count": 0,
        "isCompiled": true,
        "errors": []
    },
    "__init___5f53cb190f7f70b5109965ef06d59e91.py": {
        "stats": {
            "total_parameters": 0,
            "parameters_with_annotations": 0
        },
        "error_count": 0,
        "isCompiled": true,
        "errors": []
    },
    "__init___68b329da9893e34099c7d8ad5cb9c940.py": {
        "stats": {
            "total_parameters": 0,
            "parameters_with_annotations": 0
        },
        "error_count": 0,
        "isCompiled": true,
        "errors": []
    },
    "__main___ec49d82b59fd16b0aa7ad9e9096a35e2.py": {
        "stats": {
            "total_parameters": 0,
            "parameters_with_annotations": 0
        },
        "error_count": 0,
        "isCompiled": true,
        "errors": []
    }
}